### DISCUSSION TITLE
Is there a getting started with develpoment guide anywhere?

### DISCUSSION THREAD

**Discussion by bigredfrog** [inkstitch/inkstitch#1290] [Q&A] [answered]
If I can get to the equivalent of hello world with inkstitch and see arbitrary changes in the source code express at runtime, I am sure I can dig in and start tweaking, breaking, fixing.

However in an ubuntu VM, I did a git clone of the inkstitch code tree into the inkscape directory, but the extension is not showing up when I launch inkscape.

~/.config/inkscape/extensions$ git clone https://github.com/inkstitch/inkstitch.git

Is there anything I should be reading or an obviousim that I am missing?

Basically, how can I get a local git clone that I can run local branches and mutate on, while having that express at runtime in a launched instance of inkscape.

Thanks in advance...

I can see there is a significant difference between the normal install ( all dependancies and a collection of .inx files, compared to what is in the git clone.

So no doubt there is much to this.

Willing to read and learn, or iterate.

Just need a "start here" pointer...

**Comment by bigredfrog** [comment, marked as answer]
To answer my own question, found

https://inkstitch.org/developers/inkstitch/manual-setup/
